src/components/action-popover/action-popover-item.component.js

```diff
@@ -12,6 +12,7 @@ import {
   MenuItemFactory,
   MenuItemIcon,
   SubMenuItemIcon,
+  StyledDiv,
```

Comment:
Maybe call this something a bit more meaningful, like `StyledActionPopoverItem` or similar